src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -53,7 +53,10 @@
            :size             32
            :default-active   @selected-tab
            :data             (tabs-data watch-only?)
-           :on-change        #(reset! selected-tab %)
+           :on-change        (fn [tab]
```

Comment:
Are you sure this needs no memoization?

Reply:
@shivekkhurana there was no memoization. Do you think it needs memoization?

Reply:
I'm not sure, you need to check in the children that are getting rendered.

Reply:
@shivekkhurana fixed it! 👍